e2e/tests/create.js

```diff
@@ -0,0 +1,18 @@
+import assert from 'assert';
+import { By, until } from 'selenium-webdriver';
+import driver from '../chromeDriver';
+import createPageFactory from '../pages/CreatePage';
+
+describe('Create Page', () => {
+    const CreatePage = createPageFactory('http://localhost:8083/#posts/create')(driver);
+
+    beforeEach(async () => await CreatePage.navigate());
+
+    it('should redirect to created post', async () => {
+        await CreatePage.setTitleValue('title', 'Test title');
```

Comment:
It'd be great if the page object had a `setValues()` method expecting an object. That way you could reduce all that to 1 line